paper/paper.md

```diff
@@ -0,0 +1,181 @@
+    orcid: 0000-0002-5866-047X
+  - affiliation: Sainte-Anne Hospital Center, Université Paris Descartes
+    name: Roca, Pauline
+    orcid: 0000-0003-2089-6636
+  - affiliation: Stanford University
+    name: Blair, Ross
+    orcid: 0000-0003-3007-1056
+date: 19 February 2019
+bibliography: paper.bib
+---
+
+# Summary
+Brain imaging researchers regularly work with large, heterogeneous,
+high-dimensional datasets. Historically, researchers have dealt with this
+complexity idiosyncratically, with every lab or individual implementing their
+own preprocessing and analysis procedures. The resulting lack of field-wide
+standards has severely limited reproducibility and data sharing and reuse.
+
+To address this problem, we and others recently introduced the Brain Imaging
+Data Standard (``BIDS``; [@Gorgolewski2016-sk]): a specification meant to
+standardize the process of representing brain imaging data. BIDS is
+deliberately designed with adoption in mind; it adheres to a user-focused
+philosophy that prioritizes common use cases and discourages complexity. By
+successfully encouraging a large and ever-growing subset of the community to
+adopt a common standard for naming and organizing files, BIDS has made it much
+easier for researchers to share, re-use, and process their data
+[@Gorgolewski2017-cz].
+
+The ability to efficiently develop high-quality spec-compliant applications
+itself depends to a large extent on the availability of good tooling.
+Because many operations recur widely across diverse contexts—for example,
+almost every tool designed to work with BIDS datasets involves regular
+file-filtering operations—there is a strong incentive to develop utility
+libraries that provide common functionality via a standardized, simple API.
+
+``PyBIDS`` [@zenodo] is a Python package that makes it easier to work with BIDS
+datasets. In principle, its scope includes virtually any functionality that is
+likely to be of general use when working with BIDS datasets (i.e., that is not
+specific to one narrow context). At present, its core and most widely used
+module supports simple and flexible querying and manipulation of BIDS datasets.
+PyBIDS makes it easy for researchers and developers working in Python to search
+for BIDS files by keywords and/or metadata; to construct BIDS-valid path names
+for new files; and to validate projects against the BIDS specification, among
+other applications.
+
+In addition to this core functionality, PyBIDS also contains an ever-growing
+set of modules that support additional capabilities. At present, PyBIDS
+includes tools for (1) reading and manipulating data contained in various
+BIDS-defined files (e.g., physiological recordings, event files, or
+participant-level variables); (2) constructing design matrices and contrasts
+that support the new ``BIDS-StatsModel`` specification (for machine-readable
+representation of fMRI statistical models); and (3) automated generation of
+partial methods sections.
```

Comment:
extend with "... to be included in publications", otherwise a bit vague methods sections